build/index.html

```diff
@@ -23,16 +23,15 @@ <h2>Headers:</h2>
             <select name="language" id="lang-menu">
               <option value="English">EN</option>
               <option value="Greek">EL</option>
-              <option value="Turkish">TR</option>
-            </select>
-          </form>
+             </select>
+         </form>
          <input class="menu-btn" type="checkbox" id="menu-btn" />
          <label class="menu-icon" for="menu-btn">Menu<a><span class="material-icons carot-icon">
             arrow_drop_down
             </span></a> </label>
          <label class="service-name"> <span>Grands and Benefits</span> </label>
          <ul class="menu">
-            <li><a  href="#">Guidence</a></li>
+            <li><a  href="#">Guidance</a></li>
```

Comment:
There is an extra space here between the `a` and the `href`

Reply:
i will remove the space

Reply:
👍